pkg/report/reporter.go

```diff
@@ -110,30 +110,36 @@ func (d *defaultReporter) runRecorder(fs afero.Fs, reportFilePath string) error
 		return fmt.Errorf("error open record file: %w", err)
 	}
 
+	defer file.Close()
```

Comment:
Is it ok to discard the error or should we handle it about the same like [here](https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/8ae60db1a59345d632b9c12f11b6192bbbeedb59/cmd/monaco/integrationtest/config_rewrite.go#L71-L88)?

Reply:
Let's just log the error

Reply:
As discussed offline: Let's move the file opening to the calling function and pass in a writer & flusher